cpp/daal/include/data_management/data/internal/numeric_table_sycl_homogen.h

```diff
@@ -72,7 +72,7 @@ class DAAL_EXPORT SyclHomogenNumericTable : public SyclNumericTable
 
 #ifdef DAAL_SYCL_INTERFACE_USM
     static services::SharedPtr<SyclHomogenNumericTable<DataType> > create(const services::SharedPtr<DataType> & usmData, size_t nColumns,
-                                                                          size_t nRows, const cl::sycl::queue & queue, services::Status * stat = NULL)
+                                                                          size_t nRows, const sycl::queue & queue, services::Status * stat = NULL)
```

Comment:
Can you please change `sycl::queue` to `::sycl::queue` in this file for consistency with other DAAL sources?

Reply:
Good catch! I changed it in the latest commit, and double checked through DAAL to make sure there weren't any others.